kodev

```diff
@@ -901,6 +902,9 @@ case "${1}" in
         shift 1
         kodev-test "$@"
         ;;
+    check)
+        luacheck --no-color -q {reader,setupkoenv,datastorage}.lua frontend plugins spec
```

Comment:
Aw, I like the colors. :-P (Something to do with CI and/or Docker?)

Reply:
yeah, i just copied it from circle ci config. updated.